server/controllers/users.js

```diff
@@ -228,6 +228,6 @@ exports.updateStarred = function(req, res){
 			res.status(200).send("Favorites has been updated");
 		}
 	}
-	starred.findOneAndUpdate({username: req.body.username},{$set: {starred:req.body.starred}}, updateStar);
+	starred.findOneAndUpdate({username: req.body.username},{$push: {starred: req.body.starred}}, updateStar);
```

Comment:
What if we're unstarring a notebook or notepage? We would have to remove it from the starred array.